src/Pact/Types/Swagger.hs

```diff
@@ -44,33 +48,39 @@ import Data.Text (Text)
 declareGenericEmpty ::
   forall a d f proxy. (Generic a, Rep a ~ D1 d f, Datatype d) =>
   proxy a -> Declare (Definitions Schema) NamedSchema
-declareGenericEmpty = declareGenericSchema mempty
+declareGenericEmpty _ = declareGenericSchema mempty (Proxy @a)
 
 -- | 'ToSchema' generic implementation with provided schema.
 declareGenericSchema ::
   forall a d f proxy. (Generic a, Rep a ~ D1 d f, Datatype d) =>
   Schema -> proxy a -> Declare (Definitions Schema) NamedSchema
-declareGenericSchema s proxy = pure $ genericNameSchema defaultSchemaOptions proxy s
+declareGenericSchema s _ = pure $ genericNameSchema defaultSchemaOptions (Proxy @a) s
```

Comment:
This change mystifies me. `genericNameSchema` takes a `proxy a` argument, why do we need to ignore it and provide a different one?